src/Applications/GEOSdas_App/fvsetup

```diff
@@ -9781,12 +9795,13 @@ sub init_agcm_rc {
     $envvars{"coupled"} = $coupled;
     set_AGCM_envvars(%envvars);
 
-    $flags{"gocart_tracers"} = $gocart_tracers;
     $flags{"carma_tracers"}  = $carma_tracers;
+    $flags{"gocart_tracers"} = $gocart_tracers;
```

Comment:
Are we intentionally changing the order of the two lines here (relative to that in develop)?  Unless there's a reason for the revised order, we shouldn't modify develop

Reply:
?? I do see the change in the order of the lines when I do a direct (two-dot) comparison of develop vs. the PR's branch, see image below
<img width="2095" alt="Screenshot 2025-03-25 at 17 18 23" src="https://github.com/user-attachments/assets/2e78fc49-4394-4254-bb3d-a5e3db3ca45e" />

Reply:
will make the order of the lines same as in develop ( different order was carried over in PR from 5.41. )